include/_antic.h

```diff
@@ -33,50 +44,86 @@
 #ifndef __ANTIC_H
 #define __ANTIC_H
 
-/* Define a structure with the antic register offsets */
+/* Define a structure with the ANTIC coprocessor's register offsets */
 struct __antic {
-    unsigned char   dmactl; /* direct memory access control */
-    unsigned char   chactl; /* character mode control */
+    unsigned char   dmactl; /* (W) direct memory access control */
+
+/* Playfield modes: */
+#define DMACTL_PLAYFIELD_NONE     0x00
```

Comment:
Please move the `#define`s of possible register values out of and below the structure itself. If it's not clear by their name to which register they apply, add a comment line before.
Please do that for in all structures in all include files where applicable.